spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1,5 +1,8925 @@
+:sectnums!:
+== Preface
+
+This is the Jakarta Server Pages™ Standard Tag
+Library 2.0 (JSTL 2.0) specification, developed by the JSR-52 expert
+group under the Java Community Process. +
+See http://www.jcp.org.
+
+== Related Documentation
+
+Implementors of JSTL and authors of JSP pages
+may find the following documents worth consulting for additional
+information:.
+
+[width="100%",cols="50%,50%",]
+|===
+|Jakarta Server Pages (JSP)
+|http://java.sun.com/jsp
```

Comment:
Update link to Jakarta Server Pages project

Reply:
Thoughts on if we should use:

Reply:
The original reference is to a project/product page, so that would mean 1) is the best replacement, but actually a reference to the spec document like 2) would feel better.

Of course, since it's only text anyway, you can put them both too.